internal/connect/hwinfo.go

```diff
@@ -59,21 +62,20 @@ func getHwinfo() (hwinfo, error) {
 	hw.Cpus = result["cpus"].(int)
 	hw.Sockets = result["sockets"].(int)
 
-	hw.Hostname = hostname()
+	hw.Hostname = result["hostname"].(string)
 	hw.CloudProvider = cloudProvider()
 
 	// Include memory information if possible.
-	if mem := systemMemory(); mem > 0 {
-		hw.MemTotal = mem
-	}
+
+	hw.MemTotal = result["mem_total"].(int)
 
 	var lscpuM map[string]string
 	if hw.Arch == archX86 || hw.Arch == archARM || hw.Arch == archPPC {
 		if lscpuM, err = lscpu(); err != nil {
 			return hwinfo{}, err
 		}
-		hw.UUID, _ = uuid() // ignore error to match original
 	}
+	hw.UUID = result["uuid"].(string) // ignore error to match original
```

Comment:
We should not discard this error, imho.

Reply:
this is only until we can implement the real collection implementation.